intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+
+class Command(BaseCommand):
+    help = "Import Fake People"
```

Comment:
We may use this to import normal users too. Please change this message to reflect that.